syntax= "proto2";


enum TransactionType
{
    E_TRANSACTION_TYPE_INVALID = 0;
    E_TRANSACTION_TYPE_QUERY_ACCOUNT = 1;
    E_TRANSACTION_TYPE_CREATE_ACCOUNT = 2;
    E_TRANSACTION_TYPE_CREATE_ROLE = 3;
    E_TRANSACTION_TYPE_DEL_ROLE = 4;
    E_TRANSACTION_TYPE_ENUM_ROLE = 5;
    E_TRANSACTION_TYPE_ENTER_GAME = 6;
    E_TRANSACTION_TYPE_MAX = 7;
};